salt/metalk8s/kubectl/configured.sls

```diff
@@ -0,0 +1,14 @@
+include:
+  - .installed
+
+Create kubectl bash completion file:
+  file.managed:
+    - name: /etc/bash_completion.d/kubectl
+    - contents: __slot__:salt:cmd.run('kubectl completion bash')
+    - makedirs: True
+    - dir_mode: 755
+    - mode: 755
```

Comment:
Is this required? I think `644` should do?

Reply:
Oh yes MB just copy-pasted the dir mode but it's wrong indeed